Cargo.toml

```diff
@@ -15,6 +15,7 @@ members = [
     "forge",
     "ui",
     "utils",
+    "chisel",
```

Comment:
_(arbitrary comment location to start a thread)_

Is there a way to log the results of statements, for example in solidity shell I'll often test things like this
<img width="652" alt="image" src="https://user-images.githubusercontent.com/17163988/198701565-abe24aa7-ac4d-44c5-bc55-69d46fa3c6b2.png">

You can see solidity-shell shows the output of `1 << 4` but chisel does not

Reply:
Yeah, was planning on getting on expression evaluation over the weekend :smile: See: [checklist](https://github.com/whitenois3/foundry/tree/feat/repl/chisel#checklist).

Will require a small refactor of the current inspection logic, but shouldn't be too bad. At the moment, you have to define a variable before inspecting it, which is pretty bad devx.

Reply:
Ah sorry missed that in the checklist! Makes sense 🙂

Reply:
This should be in a fairly stable state now. Let me know if you run into any issues with it!